sylvia-derive/src/parser.rs

```diff
@@ -315,19 +315,21 @@ pub fn parse_struct_message(source: &ItemImpl, ty: MsgType) -> Option<(&ImplItem
 
 #[derive(Debug)]
 pub struct Custom<'a> {
-    msg: Path,
-    _query: Path,
+    msg: Option<Path>,
+    _query: Option<Path>,
     input_attr: Option<&'a Attribute>,
+    default_type: Path,
 }
 
 impl Default for Custom<'_> {
     fn default() -> Self {
         let sylvia = crate_module();
 
         Self {
-            msg: parse_quote!(#sylvia ::cw_std::Empty),
-            _query: parse_quote!(#sylvia ::cw_std::Empty),
+            msg: None,
+            _query: None,
             input_attr: None,
+            default_type: parse_quote! {#sylvia ::cw_std::Empty},
```

Comment:
This is not supposed to be a field - it is basically a constant. I know you cannot make it constant, but it is not something that can have other values, better to make it into `fn default_type() -> Path` and build on-demand.